test/test_endpoints.py

```diff
@@ -248,3 +249,22 @@ def test_error_response_class_by_hash():
     assert resp.status_code == 500
     expected_message = f"Class with hash {INVALID_HASH} is not declared"
     assert expected_message == error_message
+
+@devnet_in_background()
+def test_create_block_endpoint():
+    """test empty block creationn"""
+    resp = get_block_by_number({"blockNumber": "latest"}).json()
+    assert resp.get('block_hash') == GENESIS_BLOCK_HASH
+    assert resp.get('block_number') == GENESIS_BLOCK_NUMBER
+
+    resp = requests.get(f"{APP_URL}/create_block").json()
+    assert resp.get('block_number') == GENESIS_BLOCK_NUMBER + 1
+    assert resp.get('block_hash') == hex(GENESIS_BLOCK_NUMBER + 1)
```

Comment:
Can you assert the content of the newly constructed block, like that its transactions field is empty.

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/189/commits/c6fe2c6187dd85e5fb9a899399271092da460983